code/modules/reagents/chemistry_properties/prop_positive.dm

```diff
@@ -992,3 +996,37 @@
 
 /datum/chem_property/positive/anticarcinogenic/process_critical(mob/living/M, potency = 1)
 	M.take_limb_damage(POTENCY_MULTIPLIER_MEDIUM * potency)//Hyperactive apoptosis
+
+/datum/chem_property/positive/regulating
+	name = PROPERTY_REGULATING
+	code = "REG"
+	description = "The chemical regulates its own metabolization, any ammount overdosed is turned into sugar."
+	rarity = PROPERTY_COMMON
+	category = PROPERTY_TYPE_METABOLITE
+	max_level = 1
+	value = 0
```

Comment:
Why 0 value instead of just the default 2?

Reply:
oops, fixed

Reply:
I think a value of 1 is fair considering the property isn't that useful anymore